src/Microsoft.Health.Operations.Functions/Management/PurgeHistoryOptions.cs

```diff
@@ -43,4 +44,10 @@ public class PurgeHistoryOptions
     /// <value>A value cron expression</value>
     [Required]
     public string? Frequency { get; set; }
+
+    /// <summary>
+    ///  Gets or sets the collection of orchestration ids which should be skipped for cleanup
+    /// </summary>
+    /// <value>A set of one or more <see cref="string"/></value>
+    public IReadOnlyCollection<string>? InstancesToSkipPurging { get; set; } = Array.Empty<string>();
```

Comment:
I am unsure if IDs is the abstraction we want here. Should this be names instead? Or perhaps even globs?

Reply:
Changed to name